httpcore5/src/main/java/org/apache/hc/core5/http/ContentType.java

```diff
@@ -105,6 +105,11 @@
     public static final ContentType WILDCARD = create(
             "*/*", (Charset) null);
 
+    /**
+     * An empty immutable {@code NameValuePair} array.
+     */
+    public static final NameValuePair[] EMPTY_NAME_VALUE_PAIR_ARRAY = new NameValuePair[0];
```

Comment:
@arturobernalg There is no need to make it public. Please change to private.

Reply:
Ok. changed